### PR TITLE
adds readme with wireframes

### PR DESCRIPTION
**Pull request by JMarcussen**

### CODE REVIEW

**Comment by JMarcussen** [comment]
Hi @axamei , I believe I have completed the assignment, however the images are not showing when I preview. Is there a way I can fix this? Thanks!

**Comment by axamei** [comment]
@JMarcussen The format of the image reference needs to change. Change to include the folder, the curly braces should be replaced by the square brackets, and no space between the square bracket and parentheses, e.g., `Wireframe of Home Page`

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
-# INF 6420 - Intro to Web Development
+Jennifer Marcussen's INF6420 Project
+
+A portfolio site showcasing my work.
+
+## Wireframes
+
+This is where I will include visual wireframes in order to organize my portfolio site.
+
+!{Wireframe of Home Page} (Wireframes-example-01.jpg)
```
Comment (axamei) [inline README.md]: Change to `Wireframe of Home Page`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+Header - will include which page you are vieewing
+
+Navigation Menu - will include links to landing page, work, and about me pages
+
+Main Content - will include logo and a link to enter portfolio
+
+Footer - will include copyright info and contact info
+
+
+!{Wireframe of Work Page} (Wireframes-example-02.jpg)
```
Comment (axamei) [inline README.md]: Change to correct format and include folder

File: README.md [hunk 3]
```diff
@@ -1,2 +1,39 @@
...
+Header - will include which page you are vieewing
+
+Navigation Menu - will include links to landing page, work, and about me pages
+
+Main Content - will include links to portfolio pages for illustration, graphic design, and photography
+
+Footer - will include copyright info and contact info
+
+
+!{Wireframe of About Page} (Wireframes-example-03.jpg)
```
Comment (axamei) [inline README.md]: Change to correct format and include folder

**Review by axamei — CHANGES_REQUESTED** [review]

**Comment by axamei** [comment]
@JMarcussen Please make sure these changes are made before Thursday.

**Comment by axamei** [comment]
@JMarcussen I still don't see the requested changes in this branch. Please reach out to me.

**Comment by JMarcussen** [comment]
@axamei do I create a new pull request now that the changes have been made?

**Comment by axamei** [comment]
@JMarcussen make a commit with the requested changes and then push the branch. It should show up here.

**Comment by JMarcussen** [comment]
All set. Made a commit, and then pushed the branch.

**Pull request closed without merging**